tests/integration/pilot/main_test.go

```diff
@@ -52,6 +67,55 @@ values:
 			}
 			return nil
 		}).
+		Setup(func(ctx resource.Context) error {
+			var err error
+			// TODO: allow using an existing namespace to allow repeated runs with 0 setup
```

Comment:
Ill  add this in the next PR, but fwiw this was the best experience developing a test I have had. With this we can run a test in 2s, allowing for rapid iteration